### DISCUSSION TITLE
Split View Button in Nemo

### DISCUSSION THREAD

**Discussion by err0r2k** [linuxmint/discussions#337] [Ideas]
It is only possible to "Split View" with the F3 Key and the set a option to always show this. It would be great to customize Nemo with a "Split View" Button, which can be added in the Tool bar under Options.

**Comment by dayvid3** [comment]
If it's any help, I created a simple action that adds the dual pane option to the Nemo context menu.  I recently posted a few simple ones that I've made on my [GitHub](https://github.com/dayvid3/Linux_Nemo_Actions/tree/main).  I use it all the time.

**Comment by mtwebster** [comment]
Added in 6.6:
https://github.com/linuxmint/nemo/commit/a3f0a31c17bef652fb05be2bc7d9d8bb51791e4d

**Reply by dayvid3** [reply]
Thanks for the heads up!

**Reply by dayvid3** [reply]
Ah, the addition was for the toolbar, but not the context menu?
